File: OfficeHoursXamarin/OfficeHours/OfficeHours/Models/MonthModel.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace OfficeHours
{
	public class MonthModel
	{
		// unique for a user.
		public int Id { get; set; }

		public DateTime Month { get; set; }
		public TimeSpan AverageIn { get; set; }
		public TimeSpan AverageOut { get; set; }

		public List<DayModel> Days { get; set; }

		public MonthModel ()
		{
		}
	}
}

File: Angular2PoC/RoutingAndNavigation/app/log/month.model.ts
import {DateTimeModel} from './datetime.model';
import {DayModel} from './day.model';
export class MonthModel {
    constructor(public id: number,
        public month: DateTimeModel,
        public averageIn: string,
        public averageOut: string,
        public days: DayModel[]) { }
}

